src/params/index.ts

```diff
@@ -53,26 +64,104 @@ export type GetParamsByPathArgs = GetParamArgs & {recursive: boolean};
 
 export async function setParam(argv: SetParamArgs): Promise<number> {
   await configureAWS(argv);
+  const ssm = new aws.SSM();
 
-  const Name = argv.path;
+  const Name = argv.withApproval ? `${argv.path}.pending` : argv.path;
   const Value = argv.value.toString();
   const Type = argv.type;
   const Overwrite = argv.overwrite;
   const KeyId = Type === 'SecureString' ? await getKMSAliasForParameter(Name) : undefined;
-  const ssm = new aws.SSM();
   const res = await ssm.putParameter({Name, Value, Type, KeyId, Overwrite}).promise();
-  return 0;
+
+  if (argv.withApproval) {
+    console.log('Parameter change is pending approval. Review change with:');
+    console.log(`  iidy --region ${argv.region} param review ${argv.path}`);
```

Comment:
`argv.region` will often be null. Instead, `import getCurrentAWSRegion from '../getCurrentAWSRegion';`...

Reply:
e.g. ```
$ iidy param --profile sandbox set --with-approval /tavis blah2
Parameter change is pending approval. Review change with:
  iidy --region null param review /tavis
```